src/functions-and-arrays.js

```diff
@@ -1,26 +1,86 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
 
+function maxOfTwoNumbers(num1 ,num2) {
+  if (num1>num2) {
+    return num1;
+  } else if (num1===num2){
+    return num1;
+  } else if (num1<num2){
+    return num2;
+  }
+}
 
 // Iteration #2: Find longest word
-const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
-
-function findLongestWord() {}
-
 
+const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot','dsjagfdlgasjdasda']
+//console.log(words.length)
+function findLongestWord(arr) {
+  if (arr.length === 0) {
+      return null;
+  } else if (arr.length === 1) {
+    return arr[0];
+  } else {
+    let palabraMagica = ""
+    let palabraLarga = 0
+    for ( i=0 ; i<arr.length ; i++){
+//      console.log(arr[i])
+      if (palabraLarga < arr[i].length) {
+        palabraLarga = arr[i].length;
+        palabraMagica = arr[i];
+      }
+            }
+          console.log(palabraMagica)
+//           }
+//          }
+}
+}
+findLongestWord(words);
 
 // Iteration #3: Calculate the sum
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
-function sumNumbers() {}
+function sumNumbers(arr) {
```

Comment:
Aca otra alternativa 👀 
```suggestion
function sumNumbers(arr) {
  let suma = 0;
  for (value of arr) {
    suma += value;
  }
  return suma;
}
```